specs/sb/api.md

```diff
@@ -342,6 +390,29 @@ A web-friendly display name is camel-cased with spaces and punctuation supported
         }, {
           "content": "40 concurrent connections"
         }]
+      },
+      "schemas": {
+        "service_instance": {
+          "parameters": {
+            "required": [
+              "fake_required_property"
+            ],
+            "properties": {
+              "fake_required_property": {
+                "type": "string"
+              }
+            }
+          }
+        },
+        "service_binding": {
```

Comment:
`service_bindings` (plural) to match the API resource we're providing the schema for